shared/chat/conversation/notices/old-profile-reset-notice.js

```diff
@@ -4,7 +4,10 @@ import {Text} from '../../../common-adapters'
 import UserNotice from './user-notice'
 import {globalColors, globalMargins} from '../../../styles'
 
-import type {Props} from './old-profile-reset-notice'
+export type Props = {
```

Comment:
:sheep: I think it's nice to still keep the flow file separate for consistency with the rest of our codebase. I tend to look for the .flow file in my editor's quick find to jump to the type definitions.